pibooth/view/window.py

```diff
@@ -60,14 +60,16 @@ def __init__(self, title,
                         (192, 0, 224, 0, 240, 0, 248, 0, 252, 0, 254, 0, 255, 0, 255,
                          128, 255, 192, 255, 224, 254, 0, 239, 0, 207, 0, 135, 128, 7, 128, 3, 0))
 
-    def _update_foreground(self, pil_image, pos=CENTER, resize=True):
+    def _update_foreground(self, pil_image, pos=CENTER, resize=True, fullscreen=False):
```

Comment:
instead of adding a new parameter, you can use the 'pos' one.
pos=FULLSCREEN.

Reply:
ok changed